### ISSUE TITLE
How to use layer_norm or LayerNormalization with tf.keras

### ISSUE THREAD

**Issue by lijianxin520** [SciSharp/TensorFlow.NET#782]
How to use layer_norm or LayerNormalization with tf.keras?
tf.contrib.layers.layer_norm

**Comment by Oceania2018** [comment]
Added to do list.
https://www.tensorflow.org/api_docs/python/tf/keras/layers/LayerNormalization

**Comment by lindadamama** [comment]
missing layerNormalization
https://github.com/tensorflow/tensorflow/tree/v2.7.0/tensorflow/python/keras/layers/normalization

**Comment by Oceania2018** [comment]
@lindadamama It's been already implemented, here is an example: https://github.com/SciSharp/SciSharp-Stack-Examples/blob/83668294eda65e5bc73c6d7d6276c1fa5057a88a/src/TensorFlowNET.Examples/GAN/MnistGAN.cs#L70

**Comment by lindadamama** [comment]
Shouldn't BatchNormalization and LayerNormalization be the two methods?

**Comment by Oceania2018** [comment]
@lindadamama Sorry, my mistake. I will add this task in to-do-list.
https://keras.io/api/layers/normalization_layers/layer_normalization/

**Comment by Wanglongzhi2001** [comment]
Close Due to layernormalization has been developed, please reopen if you still have question, thanks!

**Comment by MattCW1701** [comment]
I know this is old, but are there any examples of actually using this? I can't add it to a Sequential Model since Sequential.Add() expects an object derived from BaseLayer and not Tensorflow.Keras.Layer.
